src/vtabs/index.js

```diff
@@ -44,6 +44,9 @@ Component({
         });
     }
   },
+  didUnmount() {
+    clearTimeout(this.timerId);
+  },
```

Comment:
`this.timerId = null;`
![image](https://user-images.githubusercontent.com/347649/42561216-32837922-852b-11e8-8a8b-7c368a2becb7.png)